src/test/java/org/opensearch/security/ConfigTests.java

```diff
@@ -76,26 +77,27 @@ public void testMigrate() throws Exception {
     public void testParseSg67Config() throws Exception {
 
         check("./legacy/securityconfig_v6/action_groups.yml", CType.ACTIONGROUPS);
-        check("./securityconfig/action_groups.yml", CType.ACTIONGROUPS);
+        check("./action_groups.yml", CType.ACTIONGROUPS);
```

Comment:
Are these changes required for this PR?

Reply:
See https://github.com/opensearch-project/security/pull/1888#discussion_r906241050